### DISCUSSION TITLE
Can't Add New Database

### DISCUSSION THREAD

**Discussion by sjarifHD** [Budibase/budibase#4150] [Help] [answered]
Hi, recently I tried to explore budibase self hosted. I Installed it at my sever along with postgres databases.
With existing postgres database I can use budibase without problem. But when I add new postgres database, it couldn't be added to apps, even I try to create new apps. The error dialog show my database doesn't exist with status code 500. I can access my database without using budibase. Any workaround I should check? Thank you....

**Comment by mjashanks** [comment, marked as answer]
Hi @sjarifHD - there should be no difference between a new and old database - Budibase supports the latest versions of postgres.

Is your database running on the same machine as Budibase? I.e. `localhost` ?

If so, then the hostname will need to be `172.17.0.1` on linux, or `host.docker.internal` on windows & mac. This is because Budibase is running inside a docker container - therefore `localhost` refers to the container itself.

**Reply by sjarifHD** [reply]
My configuration for new database is same to existing database, the different just in database name.

**Reply by nunick** [reply]
Just for anyone else: using Host: `host.docker.internal` allowed me to connect to my Postgres db. I was having the same code 500 connection error and using `localhost` or my IP wasn't working.
